src/main/java/pro/sky/homework_2_9/EmployeeController.java

```diff
@@ -6,14 +6,14 @@
 public class EmployeeController {
 
     @ExceptionHandler
-    public String handleException(Exception e){
+    public String handleException(RuntimeException e){
         return e.getMessage();
     }
 
-    private final EmployeeServiceC employeeService;
+    private final EmployeeServiceImpl employeeService;
 
     // инжектим EmployeeService в EmployeeController
-    public EmployeeController(EmployeeServiceC employeeService) {
+    public EmployeeController(EmployeeServiceImpl employeeService) {
```

Comment:
```suggestion
    public EmployeeController(EmployeeService employeeService) {
```
инъекцию всегда надо делать по интерфейсу, а не реализации